style: apply /style-guide pass to support/models/articles (#2672)
wandb/docs: support/models/articles/best-practices-to-organize-hyperparamete.mdx

keywords: ["Hyperparameter", "Sweeps", "Runs"]
---

> When you run many hyperparameter searches, tagging your runs makes it easier to group and find related experiments later.

Reviewer: ```suggestion
When you run many hyperparameter searches, tag your runs so you can group and find related experiments later.
```

Author: Applied your suggestion verbatim in e00dafe3d. ✅

> Set unique tags with `wandb.init(tags='your_tag')`. You can then filter project runs efficiently by selecting the corresponding tag in a project page's **Runs** table.

Reviewer: Separate out how to set tags in your code (the point of the article) from what it lets you do in the UI later. Maybe link to the relevant UI docs.

Author: Resolved in e00dafe3d — split into “Tag runs in your code” and “Filter tagged runs in the UI”, with links to the tags reference and Filter runs docs. ✅
